scipy/linalg/flapack_gen.pyf.src

```diff
@@ -978,6 +978,24 @@ subroutine <prefix>geqrf(m,n,a,tau,work,lwork,info)
 
 end subroutine <prefix>geqrf
 
+subroutine <prefix>geqrf_lwork(m,n,a,tau,work,lwork,info)
+    ! work, info = geqrf_lwork(m, n)
+    ! Calcualte the size of the ?geqrf work array.
```

Comment:
```suggestion
    ! Calculate the optimal size of the ?geqrf work array.
```

Reply:
I'm not quite sure how that managed to slip through spell check =/ It's fixed now.